spec/factories/users.rb

```diff
@@ -20,6 +20,17 @@
     sequence(:username) { |n| "username#{n}" }
     password 'changeme'
     password_confirmation 'changeme'
+    googleplus { Faker::Internet.url('http://plus.google.com/testosemurl') }
```

Comment:
There is a best practice for factories: Just Enough Data
https://semaphoreci.com/community/tutorials/working-effectively-with-data-factories-using-factorygirl

That means, you should only put data in your factory which is necessary to create a valid user. As you don't validate the presence of these fields you should drop it.

And furthermore, according to the [faker gem docs ](https://github.com/stympy/faker)this will produce urls like:
```http://http://linkedin.com/testosemuser/francis```